### ISSUE TITLE
pretrain NCBI BERT based on new released WWM BERT model

### ISSUE THREAD

**Issue by bugface** [ncbi-nlp/bluebert#4]
Google recently released two new BERT models with Whole Word Masking strategy (BERT-Large(Base), Uncased (Whole Word Masking)).  Do you have a plan to pre-train new NCBI models based on this new release?

**Comment by yfpeng** [comment]
We don't plan to train more models recently.
